src/app/modules/products/components/product-card/product-card.component.html

```diff
@@ -8,7 +8,15 @@ <h2 class="product-card-body-title">{{product.name}}</h2>
         <p *ngFor="let description of product.displayedDescription">{{description}}</p>
         <div class="product-card-body-price-container">
             <div class="product-card-body-price">{{product.price}}</div>
-            <button class="product-card-body-add-to-cart" (click)="onClick()">Add to cart</button>
+            <button
+                class="product-card-body-add-to-cart snipcart-add-item"
+                (click)="onClick()"
```

Comment:
snipcart takes care of the click event under the hood, so we dont need this handler